lib/pipe/csv.js

```diff
@@ -0,0 +1,110 @@
+const path = require("path");
+const fs = require("fs");
+const csvWriter = require('csv-writer');
+const { v4: uuidv4 } = require('uuid');
+const { isSameTest } = require('../util');
+const chalk = require('chalk');
+
+class CsvPipe {
+    #exportDir = process.env.EXPORT_DIR || "export"; //TODO: QUESTION: How create .env var?... and then should be removed
```

Comment:
Let's simplify pipe configuration and remove this parameter at all and use only one to save a file :arrow_down: